pkg/objstore/s3/s3.go

```diff
@@ -152,6 +157,11 @@ func NewBucketWithConfig(logger log.Logger, config Config, component string) (*B
 		sse = encrypt.NewSSE()
 	}
 
+	if config.TraceConfig.Enable {
+		logWriter := log.NewStdlibAdapter(level.Debug(logger), log.MessageKey("s3TraceMsg"))
```

Comment:
```suggestion
		logWriter := log.NewStdlibAdapter(level.Debug(logger), log.MessageKey("bucketTraceMsg"))
```

minio client can be used with minio server right?

Reply:
It *is* S3 - that's how the interface is called. Refer to https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html.

Reply:
Also, it doesn't make sense to use some kind of generic key like that since the different clients that we use have different formats of log messages. Unless, of course, you can make them the same but that is out of scope of this simple PR.